lit_tests/kernel/wave/attention/extend_attention.py

```diff
@@ -314,10 +314,11 @@ def test_causal_extend_attention_32x32x8():
 
     # CHECK-LABEL:       test_causal_extend_attention_32x32x8
```

Comment:
Can you add a separate lit test that shows what the expected IR  on a simple example?